### DISCUSSION TITLE
Option to disable bookmarks backup?

### DISCUSSION THREAD

**Discussion by zevlee** [nextcloud/bookmarks#1778] [Q&A] [answered]
Is there a way to disable Bookmarks Backups to Files? I do not find an option to do so.

I use Nextcloud Bookmarks with Floccus, so my bookmarks are saved already in my browser.

**Comment by marcelklehr** [comment, marked as answer]
There is not, currently. Also, it's worth considering, that [syncing is not a backup](https://gigaom.com/2010/03/26/syncing-does-not-a-backup-make/). If you are concerned about disk space, there will be a maximum of 15 backups at a time, covering the last 6 months.

**Reply by zevlee** [reply]
Thank you for letting me know. I understand that syncing is not a backup. However, I also regularly back up my entire Nextcloud server, including the data for apps. Therefore, the bookmarks backup is redundant for my use case.

I will create an issue requesting the ability to disable the backup feature.

**Reply by contagon** [reply]
As a followup, is there a way to specify the location of the backup? I would prefer not to have to have a "Bookmark Backups" folder in the root of my storage - I have a backups folder I'd like to put it in.

**Reply by marcelklehr** [reply]
Yes, in your Bookmarks App settings in the left bottom corner of the app you should be able to set a folder

**Reply by marcelklehr** [reply]
Note, that you can now disable backups at the same location.

**Reply by contagon** [reply]
I was driving myself crazy looking for it in the admin settings, makes much more sense that it's in the app on a per user basis. Thanks so much!

**Reply by mwllgr** [reply]
> I was driving myself crazy looking for it in the admin settings, makes much more sense that it's in the app on a per user basis. Thanks so much!

Same thing happened to me right now. I think it would be nice if we could disable backups globally, too. Or at least a small information on where to disable backups on a per-user basis in the admin settings.

**Reply by marcelklehr** [reply]
Help me understand, please: Why should backups not be a user decision? Do you want to restore database backups for your whole instance when a single user requests it, because their bookmarks are jumbled from syncing?

**Reply by mwllgr** [reply]
> Help me understand, please: Why should backups not be a user decision? Do you want to restore database backups for your whole instance when a single user requests it, because their bookmarks are jumbled from syncing?

No, it's just that some users of my Nextcloud just are not that tech-savvy and they tend to overthink when a folder called "Bookmarks Backup" suddenly appears in their root directory. This would stop these requests - also, some people just like to disable a backup feature because the corresponding data is just not that important or is backuped in another way.
